src/js/components/accordion.js

```diff
@@ -259,12 +264,12 @@ export default class Accordion {
     this.destroy();
```

Comment:
I'm pretty sure you can remove this call to `this.destroy()`. In the old version of the collapsible/accordion this made sense because there was one global instance instead of individual instances. In  this context (inside the `.init()` method) calling `this.destroy()` would have any affect because the instance you are creating when this code runs doesn't exist yet.